core/app/c/[communitySlug]/types/actions.ts

```diff
@@ -18,6 +18,22 @@ export const addPubField = defineServerAction(async function addPubField(
 	).execute();
 });
 
+export const updateTitleField = defineServerAction(async function updateTitleField(
+	pubTypeId: PubTypesId,
+	pubFieldId: PubFieldsId
+) {
+	await autoRevalidate(
+		db.updateTable("_PubFieldToPubType").set({ isTitle: false }).where("B", "=", pubTypeId)
+	).execute();
+	await autoRevalidate(
+		db
+			.updateTable("_PubFieldToPubType")
+			.set({ isTitle: true })
+			.where("A", "=", pubFieldId)
+			.where("B", "=", pubTypeId)
+	).execute();
```

Comment:
I couldn't figure out how to do this in one query :( I tried using a `with` but that didn't work, which I remember not working for stages and pubs a bit ago too...